source/src/localisation/cs_MeshTopology.cpp

```diff
@@ -70,41 +63,39 @@ uint8_t RssiDataTracker::getVarianceRepresentation(float variance) {
 	return 7;
 }
 
-// REVIEW: Mean what?
-uint8_t RssiDataTracker::getMeanRepresentation(float mean) {
-	// REVIEW: Isn't rssi negative?
+uint8_t MeshTopology::getMeanRssiRepresentation(float mean) {
```

Comment:
I was expecting an `int8_t` here, as that's the usual type for RSSI.

Reply:
To squeeze all the average rssi values, variances and samplecounts into a single mesh packet we don't have space for the sign bit. See flushAggregatedRssiData (line 128) for how these return values are used.